src/component/teacher/Teacher.js

```diff
@@ -0,0 +1,83 @@
+import { useState, useEffect } from "react";
+import RenderLesson from "./RenderLesson";
+import Form from "./Form";
+import AddQuestion from "./AddQuestion";
+import axios from "axios";
+
+
+function Teacher() {
+  const [visible, setVisible] = useState(false);
+  const [lessons, setLessons] = useState([]);
+ 
+
+  useEffect(() => {
+    axios.get("https://ldn8-islington.herokuapp.com/lessons/")
+    .then((res) => {
+      setLessons(res.data);
+    });
+  }, []);
+
+
```

Comment:
Delete line 20, I only want one space between 18 and 21.

Reply:
Thank you @douglas86. Done!